cmd/mqtt/main.go

```diff
@@ -122,6 +130,35 @@ func main() {
 		log.Fatalf(err.Error())
 	}
 
+	if cfg.mqttTargetHealthCheck != "" {
+		for i := 1; i <= maxHealthCheckRetries; i++ {
+			res, err := http.Get(cfg.mqttTargetHealthCheck)
```

Comment:
New `res` object is created on every iteration, so use `defer` to close it's body. Defers will be intelligent enough to know to which object they apply at the end of the function.